src/Chainweb/Pact/RestAPI/Orphans.hs

```diff
@@ -3,34 +3,31 @@
 
 module Chainweb.Pact.RestAPI.Orphans () where
```

Comment:
Should we move these to Pact? Are they meaningful there?

Reply:
I also have a couple of Aeson instances of GasPrice in Chainweb.Oprhans that I assume needs to be moved to Pact